documentation/release-notes/topics/version_4.rst

```diff
@@ -69,6 +69,18 @@ The following new custom components have been added to allow customers to log pe
 - Tracking Hardware
 - Generic Component
 
+==================================
+Enhanced Feature: Computed sensors
+==================================
+The Location asset type will have the following new computed sensors:
+- PUE. This is an explicit sensor in addition to the current three PUE computed sensors. This sensor is configured from the location Properties page.
+- Carbon emission equivalent.
+- Carbon emission equivalent with offset.
```

Comment:
Should this line also mention: The offset percentage is configured in the location Properties page.